.env.example

```diff
@@ -1,14 +1,14 @@
 REACT_APP_FIREBASE_API_KEY=AIzaSyB_maY9MofiRR3ECnHCRtF9J_Q4HTowBaE
 REACT_APP_FIREBASE_APP_ID=1:372391384352:web:81a59f0f888c124655f345
-REACT_APP_FIREBASE_AUTH_DOMAIN=tpr-canada.firebaseapp.com
+REACT_APP_FIREBASE_AUTH_DOMAIN=dtpr-canada.firebaseapp.com
 REACT_APP_FIREBASE_MESSAGING_SENDER_ID=372391384352
 REACT_APP_FIREBASE_PROJECT_ID=dtpr-canada
 REACT_APP_FIREBASE_STORAGE_BUCKET=dtpr-canada.appspot.com
-
-REACT_APP_EXPANDED_SURVEY=https://docs.google.com/forms/d/e/1FAIpQLSevvV3ZW9IAdJHeh3rysKxGLTi0U9XYj-AQzKHcGrHCup8CIA/viewform
-
 REACT_APP_CUSTOMER_ID=1234abcd
-
 REACT_APP_GOOGLE_FONTS_LINK=https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;600;700&display=swap
 REACT_APP_ICONS_BASE_URL=https://raw.githubusercontent.com/Helpful-Places/dtpr/master/dtpr_icons
 REACT_APP_MAPBOX_ACCESS_TOKEN=pk.eyJ1Ijoibm9ybWtlaSIsImEiOiJjanN4b254eDIwNnU2NDNxamluczhpamp6In0.uwwugNjn_-kKxLeKW0gZwg
+REACT_APP_SERVICE_ACCOUNT_AUTH_EMAIL=XXX
+REACT_APP_SERVICE_ACCOUNT_AUTH_KEY=XXX
```

Comment:
We can't expose private keys in with the `REACT_APP_` prefix, CRA has no option for private environment variables since there is no server